### ISSUE TITLE
Artist Metadata is fetch even though library setting told it not to do so

### ISSUE THREAD

**Issue by gnattu** [jellyfin/jellyfin#2217]
**Describe the bug**

Even with all Metadata source unchecked, jellyfin still fetch and update metadata of music artists

**To Reproduce**

1. Go to Library Settings, check show advanced options, uncheck all source
2. Go back to the home page
3. Right Click Music Library, then refresh metadata, select "find missing metadata"
4. Artist metadata is fetch even no source is selected

**Expected behavior**

No Artist metadata should be fetched as all sources are disabled

**Logs**

No error log, the console shows the artist metadata is been fetched

**Screenshots**

**System (please complete the following information):**
 - OS: Windows Server 2019
 - Browser: Chrome on Linux
 - Jellyfin Version: 10.4.3
 - Installed Plugins: Anime
 - Reverse proxy: N/A

**Additional context**

I have to disable this because lots of artist metadata is matched to wrong person (as they have same name).

**Comment by cvium** [comment]
This is a known issue with how artists are handled in the backend. There's unfortunately no easy fix.

**Comment by JustAMan** [comment]
No fix currently possible until https://github.com/jellyfin/jellyfin/pull/1431 lands.

**Comment by jucor** [comment]
Seems that #1431 landed three weeks ago in #2293 :) Not blocked anymore?

**Comment by JustAMan** [comment]
Not exactly... #2293 actually brought the starting point, and this journey would be much longer.
This should have to wait until we (read: @barronpm) convert library.db into EFcore...

**Comment by neo-neo1** [comment]
Experiencing this issue as well still on 10.7.7

**Comment by emes81** [comment]
I'm still experiencing this in version 10.7.7 too.

**Comment by kpfleming** [comment]
This is really frustrating, especially since the downloaded data often has the artist name in a slightly different case from what is in the media files (in my case `Alice in Chains` -> `Alice In Chains` among others), so in the 'Album Artist' view there are two listings, and the images downloaded for the artist are only displayed on the one that JF created.
